### PR TITLE
feat: rename to nRF Asset Tracker

### PR DESCRIPTION
**Pull request by coderbyheart**
- [x] Rename the `AssetTrackerCloudExample` tag to `nRFAssetTracker`

BREAKING CHANGE: this renames the project

### CODE REVIEW

**Comment by coderbyheart** [comment]
@umapraseeda I've also renamed the cloud resources to `nrf-asset-tracker` and made it so the cloud components do not have `cat-tracker` in the name (because they are generic) but use `nrf-asset-tracker`.

File: docs/aws/Index.rst [hunk 1]
```diff
@@ -1,9 +1,9 @@
 .. _index_aws:
 
-Asset Tracker Cloud Example for AWS
-###################################
+AWS cloud components 
+#####################
 
-The :ref:`Asset Tracker Cloud Example <project>` for AWS is a reference implementation of a serverless backend for an IoT product developed using `AWS CDK <https://aws.amazon.com/cdk/>`_ in `TypeScript <https://www.typescriptlang.org/>`_.
+The AWS cloud components of the :ref:`nRF Asset Tracker <project>` are a reference implementation of a serverless backend for an IoT product developed using `AWS CDK <https://aws.amazon.com/cdk/>`_ in `TypeScript <https://www.typescriptlang.org/>`_.
```
Comment (umapraseeda) [inline docs/aws/Index.rst]:
```suggestion
The nRF Asset Tracker provides a reference implementation of a serverless backend for an IoT product via the AWS cloud components.
It is developed using `AWS CDK <https://aws.amazon.com/cdk/>`_ in `TypeScript <https://www.typescriptlang.org/>`_. 
```

File: docs/device-cloud-protocol/FourKindsOfData.rst [hunk 2]
```diff
@@ -2,7 +2,7 @@
 The four kinds of data
 ######################
 
-This sections will provide a general introduction in the way devices communication with the cloud in the Asset Tracker Cloud Example project.
+This sections will provide a general introduction in the way devices communication with the cloud in the nRF Asset Tracker project.
```
Comment (umapraseeda) [inline docs/device-cloud-protocol/FourKindsOfData.rst]:
```suggestion
This section provides a general introduction on the communication of the devices with the cloud in the nRF Asset Tracker project.
```

File: docs/guides/CellGeolocations.rst [hunk 3]
```diff
@@ -44,13 +44,13 @@ This should also be the preferred aproach when developing an IoT product that ha
 If a device knows its cell tower it is mormally safe to assume that it has an internet connection in order to request the approximate location from the cloud.
 This allows to offload expensive calculations to the cloud and reduce the resource usage on the device.
 
-In the Asset Tracker Cloud Example every time a device reports a GPS fix, that position is stored together with the cell id.
+In the nRF Asset Tracker every time a device reports a GPS fix, that position is stored together with the cell id.
 Over time this builds a cell location database that is independet from a third party API and up to date.
-It is used as the primary mean of geolocation cells in the Asset Tracker Cloud Example.
+It is used as the primary mean of geolocation cells in the nRF Asset Tracker.
```
Comment (umapraseeda) [inline docs/guides/CellGeolocations.rst]:
```suggestion
It is used as the primary means of geolocated cells in the nRF Asset Tracker.
```

File: docs/guides/CellGeolocations.rst [hunk 4]
```diff
@@ -44,13 +44,13 @@ This should also be the preferred aproach when developing an IoT product that ha
...
 Only if a cell has not previously been geolocated by a device will the third-party API be called, **if it is enabled**.
 
 Geolocating cells using third-party APIs
 ========================================
 
 Third-party APIs and services like `UnwiredLabs <https://unwiredlabs.com/>`_ and `CellMapper <https://www.cellmapper.net/>`_ have a database of cell tower locations and provide an API to query against them.
-The Asset Tracker Cloud Example implements the optional resolution on the cloud side using *UnwiredLabs* on :ref:`AWS <aws-unwired-labs-api>`  for cell that have not been geolocation by devices.
+The nRF Asset Tracker implements the optional resolution on the cloud side using *UnwiredLabs* on :ref:`AWS <aws-unwired-labs-api>`  for cell that have not been geolocation by devices.
```
Comment (umapraseeda) [inline docs/guides/CellGeolocations.rst]:
```suggestion
The nRF Asset Tracker implements the optional resolution on the cloud side using *UnwiredLabs* on :ref:`AWS <aws-unwired-labs-api>` for the cells that have not been geolocated by the devices.
```

File: docs/aws/Index.rst (hunk 1 above)
Comment (coderbyheart) [reply docs/aws/Index.rst]: There is no such thing as "AWS cloud components".
AWS is a standalone term, and means "Amazon Web Services".
```suggestion
The nRF Asset Tracker provides a reference implementation of a serverless backend for an IoT product using AWS.
It is developed using `AWS CDK <https://aws.amazon.com/cdk/>`_ in `TypeScript <https://www.typescriptlang.org/>`_. 
```

File: docs/guides/CellGeolocations.rst (hunk 3 above)
Comment (coderbyheart) [reply docs/guides/CellGeolocations.rst]:
```suggestion
It is used as the primary means of geolocating cells in the nRF Asset Tracker.
```
It is about the process of acquiring the geolocation of cells, so `geolocating` sounds better to me.

**Comment by coderbyheart** [comment]
Rebased against latest saga.

**Review by umapraseeda — APPROVED** [review]

**Comment by coderbyheart** [comment]
Waiting for @joakimtoe to be added to this organization for the review.

**Comment by coderbyheart** [comment]
A doc build is available [here](https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/suites/2205386820/artifacts/45567657).

**Review by joakimtoe — APPROVED** [review]

**Pull request merged**